community/bolt/src/docs/dev/examples.asciidoc

```diff
@@ -436,3 +436,148 @@ Server: SUCCESS {
   74 69 66 69  65 72 73 91  83 6E 75 6D  00 00
 
 ----
+=== Accessing notifications
+While server executing a statement, notifications might be produced to provide some extra information for running the statement.
```

Comment:
"When Neo4j executes a statement, it may include notifications in the result stream."